test-suite/output-coqtop/attributes.out

```diff
@@ -30,6 +30,6 @@ Coq < Coq < Toplevel input, characters 30-37:
 > #[universes(polymorphic=yes, bla=bla)] Definition a := 3.
 >                              ^^^^^^^
 Error: This command does not support this attribute: universes.
-[unsupported-attributes,parsing]
+[unsupported-attributes,parsing,default]
```

Comment:
Do we really want to display the `default` category? Sure it would be harder to discover otherwise (it can still be documented in the refman) but that feels a bit cluttered.

Reply:
I like how it make `default` more discoverable.

Reply:
Ok, let's consider me the minority.